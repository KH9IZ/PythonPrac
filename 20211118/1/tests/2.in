@objcount
class QkRq:
    def __del__(self):
        print("Self destruction...")

my_petux = QkRq()
print(QkRq.__del__.__name__)
print(my_petux.counter)
del my_petux
print(QkRq.counter)
