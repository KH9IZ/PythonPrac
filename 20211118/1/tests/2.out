__del__
1
Self destruction...
0
